OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -37,6 +38,7 @@ public abstract class AttackBase : IIssueOrder, IResolveOrder, IOrderVoice, ISyn
 	{
 		[Sync] public bool IsAttacking { get; internal set; }
 		public IEnumerable<Armament> Armaments { get { return getArmaments(); } }
+		public bool IgnoresVisibility { get { return Info.IgnoresVisibility; } }
```

Comment:
Info is already public, so callers may as well access `Info.IgnoresVisibility` directly.